packages/core/src/components/forms/_input-group.scss

```diff
@@ -185,11 +185,11 @@ $input-button-height-small: $pt-button-height-smaller !default;
       @include pt-input-small();
 
       &:not(:first-child) {
-        padding-left: 25px;
+        padding-left: $pt-grid-size * 2.5;
       }
 
       &:not(:last-child) {
-        padding-right: 25px;
+        padding-right: $pt-grid-size * 2.5;
```

Comment:
this is not the correct formula. it should be `icon-size + input-small-padding * 2`. same with the one above -- it's not actually based exclusively on grid-size.

Reply:
@giladgray  I don't see `$input-small-padding` as a variable anywhere. Should I make one?

Reply:
@giladgray Can you give me the exact changes to use? I am having trouble figuring out exactly what variables to use while simultaneously making it look good.

Reply:
yes i requested that you create it earlier.